src/platform/BrowserPlatformTools.template

```diff
@@ -154,4 +154,9 @@ interface Window {
 }
 
 declare var window: Window;
-window.Buffer = require("buffer/").Buffer;
\ No newline at end of file
+if (typeof window !== "undefined") {
```

Comment:
@pleerock @daniel-lang 
I hope this change here is acceptable. I need to check if window or global exists. And if global exists instead of window, then we need to use global. 

NativeScript needs `typeorm/browser` as normal `typeorm` will not work in absence of fs, path modules. **BUT** NativeScript's top level scope is `global` and not `window`

Reply:
Would something like `getGlobalVariable().Buffer = require('Buffer/').Buffer;` work too?